README.md

```diff
@@ -142,3 +142,19 @@ To disable an openwisp service container and plug your own service like database
    - Change the [email configuration variables](docs/ENV.md) to point to your instances.
 - openwisp-nginx:
    - Configurations in `build/openwisp_nginx/` are helpful to replicate in your own instance.
+
+# Makefile Options
```

Comment:
Can you format each option as a list or something easier to read.  This section currently displays as a single paragraph.